### ISSUE TITLE
Inverting LUT in GUI for self-training

### ISSUE THREAD

**Issue by skeltoh** [MouseLand/cellpose#537]
I'm using cellpose to segment brightfield sections (DAB stained). All of the pretrained models seem to want the LUT inverted, which is easy enough to do for training data, and cyto2 in particular seems to be decent.

However, it does need some correction, so I was planning to do that and train a new model. It is very challenging to identify the cells with an inverted LUT within the cellpose GUI. Is there a way to (re)invert the LUT in the GUI, or some other way I can train manually?

**Comment by carsen-stringer** [comment]
we can add back an invert option but this will take some time to implement. if you label them in the GUI or another place, you can save the masks and then use the inverted images to train in our colab notebook: https://colab.research.google.com/github/MouseLand/cellpose/blob/main/notebooks/run_cellpose_2.ipynb

**Comment by carsen-stringer** [comment]
this is implemented on the `dev` branch, going to close this
